services/mailer/mail_team_invite.go

```diff
@@ -33,13 +33,22 @@ func MailTeamInvite(ctx context.Context, inviter *user_model.User, team *org_mod
 
 	locale := translation.NewLocale(inviter.Language)
 
+	// check if a user with this email already exists
+	user, err := user_model.GetUserByEmail(ctx, invite.Email)
+	if err != nil && !user_model.IsErrUserNotExist(err) {
+		return err
+	}
+	userAccountExists := err == nil && user != nil
```

Comment:
Hm, the user maybe inactive or disable login?

Reply:
In that case, would we want the user to be directed to sign up? Would that be an improvement over being directed to log in?

Reply:
Maybe they should not be invited and this function could just return error.